src/Controller/Admin/CustomerController.php

```diff
@@ -67,7 +67,14 @@ public function indexAction($page, Request $request)
      */
     public function createAction(Request $request)
     {
-        return $this->renderCustomerForm(new Customer(), $request);
+        $defaults = $this->getParameter('kimai.defaults')['customer'];
```

Comment:
Please use dependency injection for these parameters, instead of accessing the super-global container here:
```
        $defaults = $this->defaults;
```
and something like 
```
    public function __construct(array $defaults) {
        $this->defaults = $this->defaults;
    }
```
Please have a look at the `config/services.yaml` how to configure the values that will be passed to the constructor. As this is an array it can't be auto wired: https://github.com/kevinpapst/kimai2/blob/master/config/services.yaml#L49